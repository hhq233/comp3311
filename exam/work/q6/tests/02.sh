psql beer -c 'select second_max(IBU) from beers;'
