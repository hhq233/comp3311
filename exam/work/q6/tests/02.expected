 second_max 
------------
        111
(1 row)
